cmd/http-tracer.go

```diff
@@ -84,16 +73,24 @@ func httpTracer(h http.Handler) http.Handler {
 		}
 
 		// Create tracing data structure and associate it to the request context
-		tc := traceCtxt{}
-		ctx := context.WithValue(r.Context(), contextTraceReqKey, &tc)
+		tc := mcontext.TraceCtxt{
+			OpID:   r.Header.Get("X-Minio-Op-Id"),
```

Comment:
we could have used `x-amz-request-id` here? @vadmeste

Reply:
Yeah, seems like that would suit.